zp-relayer/configs/relayerConfig.ts

```diff
@@ -52,6 +52,7 @@ const config = {
   logHeaderBlacklist: (process.env.RELAYER_LOG_HEADER_BLACKLIST || defaultHeaderBlacklist)
     .split(' ')
     .filter(r => r.length > 0),
+  blockedCountries: (process.env.RELAYER_BLOCKED_COUNTRIES || '').split(' ').filter(r => r.length > 0),
```

Comment:
Let's validate that provided country codes are correct?

Reply:
Yeah, makes sense

Reply:
Done